script/fixes/108_source_cleaning.rb

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env ruby
```

Comment:
This seems to reimplement what you already did in the source alternates. Why not simply call `strip_source` on the post instead? Later on you can save with `post.save(validate: false)`(I think) to skip creating a post version.

This would also allow us to rerun this later on, when potential new source cleanup happens. Also makes it easier to review, since I don't have to check everything twice and removes some risk, since this code doesn't run through the tests you wrote.